Dockerfile

```diff
@@ -0,0 +1,25 @@
+# app/Dockerfile
+
+FROM python:3.9-slim
```

Comment:
Sure! That's a great idea! 

When it comes to GPU version it's not a matter of `dockerfile` I think but rather `web_checking.py` script which is run by this container. I don't think it makes sense to alter current behaviour, since working with the GPU in the javascript written by python isn't something that's straightforward to me. Additionally we would have to prepare a cloud VM which supports such operations and I'm not sure how to do that yet......

Reply:
Right, let's keep that in mind for the future then.